frontend/src/scenes/feature-flags/featureFlagLogic.ts

```diff
@@ -778,6 +792,18 @@ export const featureFlagLogic = kea<featureFlagLogicType>([
                 return Math.min(total, 100)
             },
         ],
+        filteredDashboards: [
```

Comment:
Based on the assumption that dashboardsModel will always have everything (which it does currently because we have a hardcoded limit of 1000 that covers every case for now..?)